Source/WebCore/platform/calc/CalcOperator.h

```diff
@@ -25,7 +25,9 @@
 
 #pragma once
 
+#include <cmath>
```

Comment:
One post-landing thought. Is this header included in other headers? If so we might not want to put a large function template into it.

Reply:
It is (although they're pretty fast to compile): https://searchfox.org/wubkat/search?q=CalcOperator.h&path=&case=false&regexp=false

It felt like a natural fit for the function template since it uses CalcOperator, but I'm open to other suggestions. A new file would work too, although I'm not sure what the name of that file would be.